internal/rbd/rbd_util.go

```diff
@@ -1089,7 +1111,7 @@ func (rv *rbdVolume) cloneRbdImageFromSnapshot(ctx context.Context, pSnapOpts *r
 	}
 
 	util.DebugLog(ctx, logMsg,
-		pSnapOpts, image, rv.imageFeatureSet.Names(), rv.Monitors)
+		pSnapOpts.String(), rv, rv.imageFeatureSet.Names(), rv.Monitors)
```

Comment:
`pSnapOpts` is a pointer to an rbdSnapshot, there should not be a need to call `.String()`